File: shipping-service-java/src/main/java/com/thinktecture/serverless/functions/KeepItWarm.java
package com.thinktecture.serverless.functions;

import java.time.*;
import com.microsoft.azure.functions.annotation.*;
import com.microsoft.azure.functions.*;

/**
 * Azure Functions with Timer trigger.
 */
public class KeepItWarm {
    /**
     * This function will be invoked periodically according to the specified schedule.
     */
    @FunctionName("KeepItWarm")
    public void run(
        @TimerTrigger(name = "timerInfo", schedule = "0 */9 * * * *")
        String timerInfo,
        final ExecutionContext context
    ) {
        context.getLogger().info("Java Timer trigger function executed at: " + LocalDateTime.now());
    }
}

File: shipping-service/Functions/KeepItWarm.cs
using System;
using Microsoft.Azure.WebJobs;
using Microsoft.Azure.WebJobs.Host;
using Microsoft.Extensions.Logging;

namespace Serverless
{
    public static class KeepItWarm
    {
        [FunctionName("KeepItWarm")]
        public static void Run(
            [TimerTrigger("0 */9 * * * *")]
            TimerInfo myTimer,
            ILogger log)
        {
            log.LogInformation($"C# Timer trigger function executed at: {DateTime.Now}");
        }
    }
}
